boards/shields/mchp_rnbd451_bt/doc/index.rst

```diff
@@ -0,0 +1,164 @@
+  controller. However, in the original design, this command is sent too early for the RNBD451, as
+  it takes approximately 4xx ms from POR to be ready to accept the first HCI command.
+
+  To address this timing issue, introduce a delay in the Zephyr application to enable Bluetooth.
+  This delay ensures that the **HCI reset command** is sent at the appropriate time. Add the following
+  code snippet to the application to introduce a 500 ms delay before calling ``bt_enable()``:
+
+  ``k_sleep(K_MSEC(500));``
+
+  Additionally, to prevent a voltage drop on the **extension header VCC pin**—which may cause the
+  RNBD451 to undergo a hardware reset after power-on; set **J100** to the **“BYPASS”** position. For more
+  information, refer to the `SAM E54 Xplained Pro Design Documentation (New Red PCB)`_.
+
+Hardware setting
+****************
+
+Using the SAM E54 Xplained Pro as an example, follow these steps to setup the hardware.
+
+#. Connect SAM E54 Xplained Pro Standard Extension Header (EXT1) with the mikroBUS Xplained Pro.
+#. Connect the RNBD451 to the mikroBUS Xplained Pro.
+#. Set the PWR_SEL jumper on the RNBD451 to Mikro 3V3 (to use power supplied by mikroBUS).
+#. Connect a USB cable to the Debug USB port.
+
+.. figure:: img/same54-with-rnbd451.webp
+   :align: center
+   :alt: RNBD451 Connected to ATMBUSADAPTER-XPRO and Mounted on SAM E54 Extension Header
+
+   **Figure 3: RNBD451 Connected to ATMBUSADAPTER-XPRO and Mounted on SAM E54 Extension Header**
+
+.. note::
+
+   When using a different Extension Header on the SAM E54 Xplained Pro, update the
+   ``rnbd451.overlay`` file accordingly.
+   For more details, refer to the  `SAM E54 Xplained Pro
+   User Guide`_.
+
+For more information about interfacing with the ATMBUSADAPTER-XPRO mikroBUS adapter,
+refer to the `RNBD451 Add-on Board Primary User Guide`_
+
+Sample application project configuration (prj.conf)
+***************************************************
+
+To select the HCI Host mode and the H4 driver, while disabling  the built-in controller,
+add the following configurations::
+
+   CONFIG_BT_HCI=y
+   CONFIG_BT_H4=y
+
+To disable Bluetooth Controller to Bluetooth Host flow control, which is not supported
+by RNBD451, add the following::
+
+   CONFIG_BT_HCI_ACL_FLOW_CONTROL=n
+
+To prevent the net-buffer from being drained during scan activity in the Central Multilink application, which can result in
+Controller unresponsive error, increase CONFIG_BT_BUF_EVT_DISCARDABLE_COUNT from 3 to 12.
+
+   CONFIG_BT_BUF_EVT_DISCARDABLE_COUNT=12
+
+
+Build the Applications
+**********************
+
+Navigate to the project root directory. For example, if the Zephyr workspace was setup
+following the Application Development section in the Zephyr Project Documentation,
+use the following command.
+
+``cd zephyrproject``
+
+Append the above mentioned ``prj.conf`` configurations to each ``prj.conf`` in Example #1~#5.
```

Comment:
@kartben ,

Is it already possible to define inside shield the correct `foo.conf` to automatically select the options when the shield be selected ?